src/tests/components/CourseDetails.test.js

```diff
@@ -12,43 +13,112 @@ describe("Course details", () => {
 
         expect(tagList).toBeEmptyDOMElement();
     });
+    */
 
-    it("displays award tag", () => {
+    it("displays level and award tag when both level and award are present", () => {
         const exampleCourse = {
+            level: "undergraduate",
             award: "BA (Hons)",
         };
 
         render(<CourseDetails course={exampleCourse} />);
 
-        const tag = screen.getByRole("listitem");
+        const tag = screen.getAllByRole("listitem");
 
-        expect(tag).toHaveTextContent("BA (Hons)");
-        expect(within(tag).getByTestId("tag-icon")).toHaveClass("c-icon--university");
+        expect(tag[0]).toHaveTextContent("undergraduate");
+        expect(tag[0]).toHaveTextContent("BA (Hons)");
+        expect(within(tag[0]).getByTestId("tag-icon")).toHaveClass("c-icon--mortar-board");
     });
 
-    it("displays start date tag", () => {
+    it("displays level and award tag when both only level is present", () => {
```

Comment:
Test name needs a little tidying
```suggestion
    it("displays level and award tag when only level is present", () => {
```